ProjectSettings/TagManager.asset

```diff
@@ -7,6 +7,8 @@ TagManager:
   - SwingArea
   - Death
   - ParallaxBackground
+  - Boulder
+  - Explosion
```

Comment:
Are these tags still being used? Seems the new added objects don't have those tags